brewchat/src/main/java/com/example/brewchat/activities/LoginActivity.java

```diff
@@ -22,21 +22,32 @@
 
 public class LoginActivity extends AppCompatActivity implements RegisterDialogListener,
         LoginListener{
-
+    LoginFragment loginFragment;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.login_screen);
-        EventBus.getDefault().register(this);
         if (savedInstanceState == null) {
-            Fragment fragment = new LoginFragment();
+            loginFragment = new LoginFragment();
 
             getSupportFragmentManager().beginTransaction()
-                    .add(R.id.login_screen_frame, fragment)
+                    .add(R.id.login_screen_frame, loginFragment)
                     .commit();
         }
     }
 
+    @Override
+    public void onResume() {
+        super.onResume();
+        EventBus.getDefault().register(this);
```

Comment:
I asking because I do not know what is the right place to register/unregister this library.

onStart/onStop are used in the documentation https://github.com/greenrobot/EventBus/blob/master/HOWTO.md